recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.2" %}
+{% set md5 = "1e09362a3f35d589f942359441050a6e" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  entry_points:
+    - coverage = coverage.cmdline:main
+    - coverage3 = coverage.cmdline:main
+    - coverage3.5 = coverage.cmdline:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - coverage
+
+  commands:
+    - coverage --help
+    - coverage3 --help
+    - coverage-3.5 --help
```

Comment:
So I'm kind of thinking we should get rid of these Python versioned executables. We don't tend to provide them with anything else. Maybe we shouldn't have them here.